<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arhivdialog</class>
 <widget class="QDialog" name="arhivdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Архив измерений</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QDateEdit" name="slDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="slPeriod">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Сутки</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Месяц</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btRefresh">
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btExit">
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btDel">
     <property name="text">
      <string>Удалить измерение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTableWidget" name="tbArhiv">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Дата
Время</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Итог
%</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пульпа
Пл.Нач.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пульпа
Пл.Кон.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пульпа
Осадок %</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Магнетит
Пл.Нач.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Магнетит
Пл.Кон.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Магнетит
Осадок %</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="btChart">
     <property name="text">
      <string>График</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
